src/main/java/com/ixortalk/organization/api/rest/OrganizationRestResource.java

```diff
@@ -98,4 +95,9 @@
     @PreAuthorize("permitAll()")
     @RestResource(exported = false)
     Collection<Organization> findByUsersLoginAndUsersStatus(String login, Status status);
+
```

Comment:
Seems complex, can we provide one method on the domain: `Organization.hasAdminAccess(String login)` and delegate to it from within the securityservice?  For all paths having to check this?  (Except for the query on organizations)

Reply:
Are you referring to the hasAdminAccess function? It checks if the user with the specified login has admin access to the specified user.
Will rename it to hasAdminAccessToUser

Reply:
I would suggest implementing it like you did with roles, `@securityService.hasAdminAccess(#user)` and again delegating to the same logic on organization.

Reply:
@wjans if I follow your suggestion, the user-deletion tests fail because of https://stackoverflow.com/a/34843369
haven't found a solution

Reply:
Separate issues logged: https://github.com/IxorTalk/ixortalk-organization-api/issues/26